AccessCheckoutSDK/AccessCheckoutSDK/CardView.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 /// A view managed by a `Card`
-public protocol CardView {
+public protocol CardView : class {
```

Comment:
I've had to add this because the mocking of this protocol was failing otherwise. This enforces that only classes (and not structures or enumerations) can implement this protocol.